automates/text_reading/src/test/scala/org/clulab/aske/automates/text/TestFunctions.scala

```diff
@@ -314,9 +365,92 @@ class TestFunctions extends ExtractionTest {
     testFunctionEvent(mentions, desired)
   }
 
+  // Tests from 1985-description and performance of CERES-Wheat_A_user-oriented wheat yield model
+
+  val t1i = "The primary variable influencing development rate is temperature."
+  failingTest should s"find functions from t1i: ${t1i}" taggedAs(Somebody) in {
+    val desired = Seq(
+      "development rate" -> Seq("temperature") //note: overlap between trigger and output??
+    )
+    val mentions = extractMentions(t1i)
+    testFunctionEvent(mentions, desired)
+  }
+
+  // Tests from paper: PT-1972-On the Assessment of Surface Heat Flux and Evaporation using Large-Scale Parameters-petpt
+
+  val t1j = "The rate of conduction of heat near the interface between two media is governed by the quantity pc√K pertaining to each medium."
+  failingTest should s"find functions from t1j: ${t1j}" taggedAs(Somebody) in {
+    val desired = Seq(
+      "The rate of conduction of heat near the interface between two media" -> Seq("quantity pc√K pertaining to each medium") //todo: identifier within Phrase should be filtered out
+    )
+    val mentions = extractMentions(t1j)
+    testFunctionEvent(mentions, desired)
+  }
+
+  val t2j = "However, when the thermal balance between the two media is disturbed, as by advection, the heat subsequently passing from one to the other over any not-too-short time is governed by and proportional to the conductive capacity of the lower ranking medium."
+  failingTest should s"find functions from t2j: ${t2j}" taggedAs(Somebody) in {
+    val desired = Seq(
+      "heat subsequently passing from one to the other over any not-too-short time" -> Seq("conductive capacity of the lower ranking medium"), //todo: tuning expansion handler needed?
+      "heat subsequently passing from one to the other over any not-too-short time" -> Seq("conductive capacity of the lower ranking medium")
+    )
+    val mentions = extractMentions(t2j)
+    testFunctionEvent(mentions, desired)
+    val functionMentions = mentions.filter(_.label == "Function")
+    functionMentions.head.text shouldEqual functionMentions.last.text
+  }
+
+  val t3j = "T0 is itself strongly governed by R and ground dryness."
+  passingTest should s"find functions from t3j: ${t3j}" taggedAs(Somebody) in {
+    val desired = Seq(
+      "T0" -> Seq("R", "ground dryness") //todo: mention with "itself" as an output should be filtered out
```

Comment:
This seems to be resolved? (I am trying this example in your branch in the web app)

Reply:
Oh, sorry, that's me forgot to delete the note again. That issue was addressed by the filterFunctionArgs action.